src/server/game/World/World.cpp

```diff
@@ -1350,18 +1350,21 @@ void World::SetInitialWorldSettings()
     ///- Init highest guids before any table loading to prevent using not initialized guids in some code.
     sObjectMgr->SetHighestGuids();
 
-    ///- Check the existence of the map files for all races' startup areas.
-    if (!MapManager::ExistMapAndVMap(0, -6240.32f, 331.033f)
-        || !MapManager::ExistMapAndVMap(0, -8949.95f, -132.493f)
-        || !MapManager::ExistMapAndVMap(1, -618.518f, -4251.67f)
-        || !MapManager::ExistMapAndVMap(0, 1676.35f, 1677.45f)
-        || !MapManager::ExistMapAndVMap(1, 10311.3f, 832.463f)
-        || !MapManager::ExistMapAndVMap(1, -2917.58f, -257.98f)
-        || (m_int_configs[CONFIG_EXPANSION] && (
-            !MapManager::ExistMapAndVMap(530, 10349.6f, -6357.29f) ||
-            !MapManager::ExistMapAndVMap(530, -3961.64f, -13931.2f))))
-    {
-        exit(1);
+    if (!sConfigMgr->isDryRun())
+    {
+        ///- Check the existence of the map files for all races' startup areas.
```

Comment:
startup areas? rather starting zones, no ?

and just from reading the comment, what about using custom maps ? (i know we only do blizzlike but just curiosity)

Reply:
I think it checks only starting areas because it's the minimum needed to run the game.

Reply:
@BarbzYHOOL I haven't modified that comment, just moved it inside an `if` block. So this is not related with this PR.

Reply:
can still improve it damnit, these comments are bad

Reply:
we can improve many things but not in this PR :)